retail/interactive-tutorial/RetailProducts.Samples/AddFulfillmentPlacesSample.cs

```diff
@@ -0,0 +1,95 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_add_fulfillment_places]
+
+using Google.Cloud.Retail.V2;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+/// <summary>
+/// The add fulfillment places sample class.
+/// </summary>
+public class AddFulfillmentPlacesSample
+{
+    /// <summary>
+    /// Get the add fulfillment palces request.
+    /// </summary>
+    /// <param name="productName">The actual name of the retail product.</param>
+    /// <returns>Add fulfillment places request.</returns>
+    private static AddFulfillmentPlacesRequest GetAddFulfillmentRequest(string productName)
+    {
+        // The request timestamp
+        DateTime requestTimeStamp = DateTime.Now.ToUniversalTime();
+
+        // The outdated request timestamp
+        // requestTimeStamp = DateTime.Now.ToUniversalTime().AddDays(-1);
+
+        AddFulfillmentPlacesRequest addFulfillmentRequest = new AddFulfillmentPlacesRequest
+        {
+            Product = productName,
+            Type = "pickup-in-store",
+            AddTime = Timestamp.FromDateTime(requestTimeStamp),
+            AllowMissing = true,
+            PlaceIds = { "store2", "store3", "store4" }
+        };
+
+        Console.WriteLine("Add fulfillment places request:");
+        Console.WriteLine($"Product Name: {addFulfillmentRequest.Product}");
+        Console.WriteLine($"Type: {addFulfillmentRequest.Type}");
+        Console.WriteLine($"Add Time: {addFulfillmentRequest.AddTime}");
+        Console.WriteLine($"Fulfillment Places: {addFulfillmentRequest.PlaceIds}");
+
+        return addFulfillmentRequest;
+    }
+
+    /// <summary>
+    /// Call the Retail API to add fulfillment places.
+    /// </summary>
+    /// <param name="productName">The actual name of the retail product.</param>
+    public static void AddFulfillment(string productName)
+    {
+        var addFulfillmentRequest = GetAddFulfillmentRequest(productName);
+        var operation = ProductServiceClient.Create().AddFulfillmentPlaces(addFulfillmentRequest);
+
+        Console.WriteLine("\nAdd fulfillment places. Please, wait.\n");
+        
+        operation.PollUntilCompleted();
+    }
+}
+// [END retail_add_fulfillment_places]
+
+/// <summary>
+/// The add fulfillment places tutorial class.
+/// </summary>
+public static class AddFulfillmentPlacesTutorial
+{
+    [Runner.Attributes.Example]
+    public static void PerformAddFulfillment()
+    {
+        string projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+
+        // Create product.
+        Product createdProduct = CreateProductSample.CreateRetailProductWithFulfillment(projectId);
+
+        // Add fulfillment info to created product.
+        AddFulfillmentPlacesSample.AddFulfillment(createdProduct.Name);
+
+        // Get product.
+        GetProductSample.GetRetailProduct(createdProduct.Name);
```

Comment:
Same comment as before, maybe write something that specifically shows what was changed, else getting the product here seems pointless?

Reply:
We have in console output created product with full info, add fulfillment request with full info and GetRetailProduct() displays product info after changes. I think we can just compare product info before changes and after. Despite the previous console output add another one that describes more detail?

Reply:
Aknowledged